### PR TITLE
Completed the Array Class, I cant get my tests to run.

### PR DESCRIPTION
**Pull request by Tpirtle**
I could not get my Tests to run.

### CODE REVIEW

File: Assignment6/Assignment6.csproj [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <ApplicationIcon />
+    <OutputType>Library</OutputType>
+    <StartupObject/>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="MSTest.TestFramework" Version="2.0.0" />
+  </ItemGroup>
```
Comment (ashermancinelli) [inline Assignment6/Assignment6.csproj]: I don't think you need the mstest framework in your library project, only the test project

File: Assignment6.Tests/Assignment6.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
```
Comment (AustinHall7798) [inline Assignment6.Tests/Assignment6.Tests.csproj]: I'm not sure if it's an issue on your project, or my own packages, but after I updated the NuGet packages on my own machine, I was able to run your tests.

Comment (whereisdaniru) [reply Assignment6.Tests/Assignment6.Tests.csproj]: ^This helped me. I was having trouble getting my tests to run also and updating the NuGets fixed part of my issue.

**Comment by MarkMichaelis** [comment]
##  (Tpirtle)

## Assignment6

| Build/Compile (Build Succeeded: 0 Error(s), 0 Warning(s))| **10** |
| --------------------- | ------ |
| Assignment compiles even though major pieces might be missing, or there are significant problems: **✔** |  **5** |
| The assignment contains only a few required elements or has significant problems. | **5** |


| Assignment  | **5** |
| --------------------- | ------ |
| Supports collecting a fixed number of items once instantiated. | ❌ |
| Has a constructor via which you can specify fixed width.  (Other constructors are welcome.) | ✔ |
| Handle null appropriately when items are missing (this is likely a judgement call so justify your answer in the comments.) | ✔ |
| Throw an exception if an item doesn't exist. | ✔ |
| Include an index operator. | ❌ |
| Support using foreach over the items in the `Array<T>` class. | ✔ |
| Support a collection initializer. | ✔ |
| Implement `System.Collections.Generic.ICollection<T>`. | ✔ |
| Ignore support for multi-dimensional arrays. | ✔ |
| Add a `Capacity` property to return the capacity (the value specified in the constructor) of the collection. | ✔ |


| Project Analysis |   |
| -----------------| - |
| Nullable Enabled: | **❌** |
| CodeReview Performed | **✔** |
| Add the code analysis packages and appropriately handle all warnings* |  **❌** |
*<sub><sup>`IntelliTectAnalyzer.dll`,`Microsoft.NetCore.Analyzers.dll`,`Microsoft.CodeQuality.Analyzers.dll`,`Microsoft.NetCore.Analyzers.dll`,`Microsoft.NetFramework.Analyzers.dll`. (It is okay to appropriately disable rules such as  localization/culture rules.)</sub></sup>


| ProjectName | Nullable | Solutions | Analyzers | Disabled Warnings | Waning Level |
| -- | -- | -- | -- | -- | -- |
| Assignment6 | Disable | Assignment6.sln |  | CS1701, CS1702, CS2008 | 4 |
| Assignment6.Tests | Disable | Assignment6.sln |  | CS1701, CS1702 | 4 |


| Testing | **5** |
| --------------------- | ------ |
| Test Run FAILED due to out of date packages | **0%** |
| Unit test project fails to compile, most test fails, or contains significant problems. | **5** |
| Most assigned work is tested or contain some errors. | **0** |
| All assigned work is tested | **0** |


| Extra Credit | **0** |
| --------------------- | ------ |
| Write a cast operator to a C# array or a `System.Collections.Generic.List<T>`. | ❌ |
| Add support for an `IReadOnlyArray` interface (for you to define) that provides readonly access to the count and each element in the array.  Be sure to appropriately decorate it for covariance and contravariance | ❌ |

**Pull request closed without merging**